workflow/rocoto/gfs_tasks.py

```diff
@@ -554,16 +554,36 @@ def _fcst_cycled(self):
     def post(self):
         add_anl_to_post = False
         if self.app_config.mode in ['cycled']:
-            add_anl_to_post = True
+            add_anl_to_post = False
```

Comment:
Should we just take this option out of `_get_postgroups()` entirely now?

Reply:
@WalterKolczynski-NOAA I will default to both yours and @aerorahul's preferences. You are far more familiar of any breaks this might induce.

Reply:
Judging by the thumb-up from Rahul, I'm saying yes go ahead and remove the option.

Reply:
Missed the thumbs up earlier; change made.

Reply:
Sorry, I think you need to either back the change back out or repeat separating the analysis for ocean post. I searched the file for `add_anl_to_post` (you missed a couple) and found that `ocnpost` is still trying to use the setting.

Reply:
@WalterKolczynski-NOAA What do I need to change back?

Reply:
There should be no `add_anl_to_post` option after this PR.

Reply:
K, then the ocnpost metatask will need to similarly be updated to have a separate oncpost for anl.

Reply:
correct. And it will, when we have any ocean analysis to post process.